modules/lazyload/module.php

```diff
@@ -0,0 +1,101 @@
+<?php
+namespace Elementor\Modules\LazyLoad;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Element_Base;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+
+	const EXPERIMENT_NAME = 'e_lazy_load_background_images';
+
+	public function get_name() {
+		return 'lazyload';
+	}
+
+	public static function get_experimental_data() {
+		return [
+			'name' => static::EXPERIMENT_NAME,
+			'title' => esc_html__( 'Lazy Load Background Images', 'elementor' ),
+			'tag' => esc_html__( 'Performance', 'elementor' ),
+			'description' => esc_html__( 'Lazy loading images that are not in the viewport improves initial page load performance and user experience. By activating this experiment all background images except the first one on your page will be lazy loaded to improve your LCP score', 'elementor' ),
+			'release_status' => Experiments_Manager::RELEASE_STATUS_BETA,
+			'default' => Experiments_Manager::STATE_INACTIVE,
+		];
+	}
+
+	public function enqueue_scripts() {
+		wp_enqueue_script(
+			'elementor-lazyload',
+			$this->get_js_assets_url( 'lazyload' ),
+			[ 'elementor-frontend' ],
+			ELEMENTOR_VERSION,
+			true
+		);
+	}
+
+	public function enqueue_styles() {
+		wp_enqueue_style(
+			'elementor-lazyload',
+			$this->get_css_assets_url( 'modules/lazyload/frontend' ),
+			[],
+			ELEMENTOR_VERSION
+		);
+	}
+
+	private function get_element_background_image_url( $settings, $keys = [ 'background_image', 'url' ] ) {
+		return Utils::get_array_value_by_keys( $settings, $keys );
+	}
+
+	private function update_element_attributes( Element_Base $element ) {
+		$settings = $element->get_settings_for_display();
+		$background_image_url = $this->get_element_background_image_url( $settings ) ?? $this->get_element_background_image_url( $settings, [ '_background_image', 'url' ] );
```

Comment:
Find by configuration, and let 3rd party define their settings (find by controls with the `background_lazyload` config like here https://github.com/elementor/elementor/pull/19706/files#diff-84ad472d82bf3c745014481fcf93736afaa03f443203f9a3ebc499fc22b3fc31R601 )

Reply:
Done, I hope I understood your request. What I did was create a config in which lazyload is set to true. By using properity, you can change the keys to get the background image.